src/config/chis-civ/config.json

```diff
@@ -0,0 +1,199 @@
+{
+  "domains": [
+    {
+      "friendly": "MoH CIV DEV",
+      "domain": "moh-civ.dev.medicmobile.org"
+    },
+    {
+      "friendly": "MoH CIV UAT",
+      "domain": "moh-civ-uat.dev.medicmobile.org"
+    },
+    {
+      "friendly": "MoH CIV PROD",
+      "domain": "moh-civ.app.medicmobile.org"
+    }
+  ],
+  "contact_types": [
+    {
+      "name": "c40_supervision_area",
+      "friendly": "Supervision Area",
+      "contact_type": "c42_supervision_area",
+      "user_role": ["supervisor", "dedicated_supervisor"],
+      "username_from_place": false,
+      "deactivate_users_on_replace": false,
+      "hierarchy": [
+        {
+          "friendly_name": "Disctrict",
+          "property_name": "district",
+          "contact_type": "c30_district",
+          "type": "name",
+          "required": true,
+          "level": 1
+        }
+      ],
+      "replacement_property": {
+        "friendly_name": "Affected Supervision Area",
+        "property_name": "replacement",
+        "type": "string",
+        "parameter": ["\\(*Supervision*\\)"],
+        "required": true
+      },
+      "place_properties": [
+        {
+          "friendly_name": "Supervision Area Name",
+          "property_name": "name",
+          "type": "generated",
+          "parameter": "Zone de supervision de: {{ contact.last_name }} {{ contact.first_name }}",
+          "required": false
+        }
+      ],
+      "contact_properties": [
+        {
```

Comment:
Should you ask for first name before last? Like how it is said?

Reply:
Done